sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -139,6 +139,7 @@ public void map(SpanData span, Consumer<TelemetryItem> consumer) {
             consumer);
     }
 
+    // TODO looks like this method can be private
     public TelemetryItem map(SpanData span, long itemCount) {
```

Comment:
this method is used in agent.

Reply:
can you show me where?

Reply:
[AgentSpanExporter line 62](https://raw.githubusercontent.com/microsoft/ApplicationInsights-Java/main/agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/AgentSpanExporter.java#:~:text=mapper.map(span%2C%20telemetryItemConsumer)%3B)

Reply:
that's not the same method